index.browser.js

```diff
@@ -27,6 +27,7 @@ module.exports = function (size) {
   var id = ''
   var bytes = crypto.getRandomValues(new Uint8Array(size))
   while (size--) {
```

Comment:
Let's make a note that `while (size--) {` is a more compact alternative for `for (var i = 0; i < size; i++)`

Reply:
Added ✅